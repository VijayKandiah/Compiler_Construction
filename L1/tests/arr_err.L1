(:main
	(:main
		0 0
		rdi <- 15
		rsi <- 11
		call allocate 2
		rsi <- 123
		rdi <- rax
		call array-error 2
		return))