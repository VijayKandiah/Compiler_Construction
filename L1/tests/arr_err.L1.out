attempted to use position 61 in an array that only has 7 positions
